### DISCUSSION TITLE
OLED 128x64 Not working

### DISCUSSION THREAD

**Discussion by wrightmac** [outdoorbits/little-backup-box#167] [Q&A]
Hello. I have a RPi 3B+, RPi OS, and LBB. 
Everything works great. I am honoring to add the screen for added feedback as to what is going on. 
The screen is connected per the docs and confirmed with a diagram of the pin outs. 
I used radio-config to turn on I2C. Rebooted. 
went into LBB and turn on the display option. 
I toss a card in, I watch the log via the web, but the screen is blank. 
Any suggestions or other info that might help. 
TIA
-wM

**Comment by outdoorbits** [comment]
Hi, radio-config? Is this auto-correction of raspi-config? ;-)
i2c should already have been activated by the installer-script. Could you please post the output of
`sudo i2cdetect -y 1 `?

**Reply by wrightmac** [reply]
0  1  2  3  4  5  6  7  8  9  a  b  c  d  e  f
00:                         -- -- -- -- -- -- -- --
10: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --
20: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --
30: -- -- -- -- -- -- -- -- -- -- -- -- 3c 3d 3e --
40: 40 41 -- -- -- -- -- -- -- -- -- -- -- -- -- --
50: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --
60: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --
70: -- -- -- -- -- -- -- --

No address shows up with 5v, but does with 3.3V. See above.  Still nothing shows up on the display. 
I’ve double-checked i2c connections. 
it is enabled in config.txt
There is nothing in /etc/modules 
It is on a Model 3 B+. 
Thank you for any help you can provide.

**Comment by outdoorbits** [comment]
The address 3c is the only one to expect for me. Are there other i2c devices connected? Could you provide at link to your display?

**Comment by wrightmac** [comment]
Sorry about the delay. 

It was one of a batch of 5 screens I bought a way back. After a little testing that screen didn’t work with the Pi. It worked on an Arduino. 

I grabbed another and all is working. 

This is a most excellent project!!!
